plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/model/CodeModernizerArtifact.kt

```diff
@@ -100,7 +99,7 @@ open class CodeModernizerArtifact(
                 throw RuntimeException("Expected root for patches was not a directory.")
             }
             return patchesDir.walk()
-                .map { fileSystem.findFileByNioFile(it) ?: throw RuntimeException("Could not find patch") }
+                .map { fileSystem.refreshAndFindFileByNioFile(it) ?: throw RuntimeException("Could not find patch") }
```

Comment:
Zongren tested and it worked

Reply:
Note: This is the main fix for "refresh VFS if diff.patch not found".